packages/mobile/src/send/SendAmount.tsx

```diff
@@ -208,7 +209,9 @@ export class SendAmount extends React.PureComponent<Props, State> {
   getConfirmationInput = () => {
     const amount = parseInputAmount(this.state.amount)
     const recipient = this.getRecipient()
-    const recipientAddress = getRecipientAddress(recipient, this.props.e164NumberToAddress)
+    const recipientAddress = recipient.address
```

Comment:
It's a bit confusing but we shouldn't use the address on the recipient itself, in case that address is out of date. We refresh the address mappings in fetchLatestPhoneAddress below.

We should use the `getRecipientAddress` helper to retrieve the latest address from the mapping.

I expect more people may be confused by this in the future though. How can we prevent people making this mistake? Maybe we should remove the address property from recipients? Only QR code and Address type recipients should have those fields.